modules/document_repository/php/files.class.inc

```diff
@@ -317,7 +317,7 @@ class Files extends \NDB_Page
                 'No file uploaded'
             );
         }
-        $filename = $uploadedFile->getClientFilename();
+        $filename = urldecode($uploadedFile->getClientFilename());
```

Comment:
@driusan not sure if this is still necessary with the use of the file uploader class

Reply:
also not sure if the decode should happen IN the file uplodaer class

Reply:
23: https://github.com/aces/Loris/pull/7383
main: https://github.com/aces/Loris/pull/5234